RELEASES.md

```diff
@@ -77,6 +77,24 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
         
     c. Create a PR on github into the release branch. Ask in the UA channel on mattermost for review.
 
+    d. When reviewing the release PR, please use the following guidelines when reviewing the new changelog entry:
+
+       * Is the version correctly updated ? We must ensure that the new version on the changelog is
+         correct and it also targets the latest Ubuntu release at the moment.
+       * Is the entry useful for the user ? The changelog entries should be user focused, meaning
+         that we should only add entries that we think users will care about (i.e. we don't need
+         entries when fixing a test, as this doesn't provide meaningful information to the user)
+       * Is this entry redundant ? Sometimes we may have changes that affect separate modules of the
+         code. We should have an entry only for the module that was most affected by it
+       * Is the changelog entry unique ? We need to verify if the changelog entry is not already
+         reflected on early version of the product. If it is, we need only need to remove but double
```

Comment:
on early versionof the product -> on an early version of the changelog
need only need to -> need not only to